### PR TITLE
Feat: vision draft v1

### PR DESCRIPTION
**Pull request by brandtkeller**
Establishing project repository for bootstrapping Proof of Concept. 

Initial draft for vision created. Opening for review early to allow iterations.

### CODE REVIEW

File: VISION.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Vision
+
+The primary focus for this project is to provide an extensible tool that can combine compliance rule-sets (or standards) with automation to facilitiate automated-auditing and real-time reporting for systems (infrastructure, applications, and other configurations). Through the utilization of the Open Security Controls Assessment Language (OSCAL), standards documents and/or other rule-sets can define when a given system is compliant, and automated tooling can perform point-in-time or continuous auditing that reports on the current state of provisioned or deployed systems.
+
+Workflow for execution would include a model where OSCAL documents existed for declaring a universe of controls that _can_ be satisfied for a given system. These documents would include (where applicable) _when_ the control is statisfied and, when ingested by the tool, the automation would perform validation that a control is or is-not compliant. Reporting would be generated in OSCAL for visualization in GRC tooling. Extensiblity is at the forefront for this project, as compliance is established across many technologies (Cloud/Infrastructure/application/etc).
```
Comment (austenbryan) [inline VISION.md]: Can we use the same language in the first and second paragraph? Infrastructure, platform, applications, etc.

Ideally this extends beyond cloud so want to be as generic as possible in the vision statement

File: VISION.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# Vision
+
```
Comment (austenbryan) [inline VISION.md:3]: I think we need to start with a problem statement before jumping to project vision. Here is a stab for iteration

"Federally regulated Information Technology (IT) systems require high standards of cyber security in order to protect critical data for its users.  Today, verifying cyber security standards is done in a waterfall process after software is built and deployed by personnel that are far removed from the creation of the system.  This current process delays software delivery by months to years, delays feedback loops on cyber security for engineers requiring that learning, and lowers morale of teams who have to do compliance work after the fact."

File: VISION.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+This is impactful to many environments and workflows that ultimately determine the security of which compliance is established. For environments that must maintaiin high-levels of compliance with standards documents (for security), this may be automation that facilitates automating (where possible) auditing/reporting of documents. While the contributing factor for developing this tooling is to automate systems against existing standards - it has been discussed how this can be extended to support systems still under development. Given that the tool executes from inputs declared in OSCAL, per conditions established in the controls, compliance can mean existing documents as well as rule-sets that are internal to teams or enterprises that develop infrastructure, platforms, applications to specific standards. These may include platforms as well as applications. For single-pane visualization, this can include controls that are non-automatable and providing a mechanism for version-controlling a given report for auditing purposes.
```
Comment (austenbryan) [inline VISION.md]: After this section we need to tie things into what Michaela is specifically looking for in her FY23 budget for OSCAL.

"There are many tools in the ecosystem today asserting OSCAL statements, compliance, and effectiveness for IT systems.  There is a need to consistently evaluate these tools for accuracy and give the community a place to discuss issues openly.  Automating workflows for OSCAL validation and reporting that information in a centralized fashion for community collaboration will drive OSCAL adoption forward ultimately decreasing software delivery times and increasing cyber security effectiveness.

File: VISION.md (hunk 1 above)
Comment (brandtkeller) [reply VISION.md]: Agreed - Shooting for generic and not wanting to spell it out each time. Is the first declaration:

> ... reporting for systems (infrastructure, applications, and other configurations)

sufficient for articulating the breadth?

File: VISION.md (hunk 3 above)
Comment (brandtkeller) [reply VISION.md]: Appended

**Review by austenbryan — APPROVED** [review]

**Review by austenbryan — APPROVED** [review]

**Pull request merged**